frame/staking/src/lib.rs

```diff
@@ -980,7 +1092,8 @@ decl_module! {
 					ledger.active = Zero::zero();
 				}
 
-				let era = Self::current_era() + T::BondingDuration::get();
+				// Note: in case there is no current era it is fine to bond one era more.
+				let era = Self::current_era().unwrap_or(0) + T::BondingDuration::get();
```

Comment:
I question why this value would be `None` and why we would `unwrap_or(0)` here.

Reply:
before the first era is planned by the first call of SessionManager::new_session one user can bond and unbond.

in case the user unbond himself then this unbond will be unlocked one era more as if he has unbound in the first era.

Reply:
It makes sense to me if we you call unbond, and `current_era` is `none`, you should be able to withdraw unbonded immediately.